src/App.tsx

```diff
@@ -15,36 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FILD_ALPHABETICCALY = 'alphabeticcaly';
```

Comment:
There is a typo in the constant name `SORT_FILD_ALPHABETICCALY`. It should be `SORT_FIELD_ALPHABETICALLY` to correctly represent the purpose of the constant.